tests/mocha/block_test.js

```diff
@@ -424,19 +424,24 @@ suite('Blocks', function() {
     });
 
     suite('Deserialization', function() {
+      setup() {
+        this.deserializationHelper = function(text) {
+          let dom = Blockly.Xml.textToDom(text);
+          Blockly.Xml.appendDomToWorkspace(dom, this.workspace);
+          this.assertConnectionsEmpty();
+          this.clock.runAll();
+        }
+      }
```

Comment:
I didn't catch this my first pass, 
you should use:
```
setup(function() {
...
});
```

And also this.deserializationHelper is missing semicolon, I think.

Same issue with the other setup, though I'd like for you to move that one to `test_helpers.js`

Reply:
Done. I also move the helper to test_helpers, then used in in place of a similar function that was used in collapsing and expanding tests.